data-sets-tests/src/test/java/org/zowe/unix/files/tests/UnixFilesDeleteFileTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2019
+ */
+package org.zowe.unix.files.tests;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+import org.apache.http.HttpStatus;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.zowe.api.common.errors.ApiError;
+import org.zowe.tests.AbstractHttpIntegrationTest;
+import org.zowe.unix.files.exceptions.FileNotFoundException;
+import org.zowe.unix.files.exceptions.NotAnEmptyDirectoryException;
+import org.zowe.unix.files.exceptions.UnauthorisedFileException;
+import org.zowe.unix.files.model.UnixDirectoryAttributesWithChildren;
+import org.zowe.unix.files.model.UnixDirectoryChild;
+import org.zowe.unix.files.model.UnixEntityType;
+
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+
+
+public class UnixFilesDeleteFileTest extends AbstractHttpIntegrationTest {
```

Comment:
Switch to extended AbstractUnixFilesIntegrationTest as that will remove the need to define UNIX_FILES_ENDPOINT, TEST_DIRECTORY and set the RestAssured basePath here

Reply:
removed and changed extending abstract class